spec/order_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'order'
+
+describe Order do
```

Comment:
After injecting Menu into Order, please use test doubles for the `Menu` class in your Order tests.

Step 19 of Boris Bikes has some examples: https://github.com/makersacademy/course/blob/master/boris_bikes/19_isolating_tests_with_doubles.md